docs/gatsby-node.js

```diff
@@ -10,7 +8,7 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
   createNodeField({
     node,
     name: "collection",
-    value: parent.sourceInstanceName,
+    value: parent.relativeDirectory.split("/")[0],
```

Comment:
I'm not sure of the point of this change? 

It might help if I were more certain of the point of the collection field in general that @silvenon added. Are we planning on sourcing different MDX? When is the filtering planned and why is a regex by file path more awkward? Right now, no filtering is necessary that I can see.